.github/workflows/linux-eic-shell.yml

```diff
@@ -619,6 +619,21 @@ jobs:
       fail-fast: false
       max-parallel: 4
     steps:
+      - name: Hold off to avoid rate limiting
+        run: |
+          rate=$(gh api rate_limit | jq '.rate')
+          echo $rate
+          limit=$(echo $rate | jq '.limit')
+          used=$(echo $rate | jq '.used')
```

Comment:
Unused, but keeping around as documentation since these are the four fields in the rate output.

Reply:
I guess the `echo $rate` kinda covers that too...

```suggestion
```